### PR TITLE
MINT-1707, MINT-1712 update mnemonic and private key import with getting profile

### PR DESCRIPTION
**Pull request by ex1st0r**

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -43,6 +43,6 @@ export { default as CopyDialog } from './dialogs/CopyDialog/CopyDialog'
 
 export { default as WalletWidget } from './wallet/WalletWidget/WalletWidget'
 export { default as Button } from './common/ui/Button/Button'
-export { default as UserRow } from './common/ui/UserRow/UserRow'
+export { default as UserRow } from '../../packages/login-ui/components/UserRow/UserRow'
```
Comment (ozalexo) [inline src/components/index.js]: export { default as UserRow } from '@chronobank/login-ui/components/UserRow/UserRow'

File: src/components/login/WalletImportPage/WalletImportPage.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ import {
 import { SubmissionError } from 'redux-form'
 import profileService from '@chronobank/login/network/ProfileService'
 import { getAddress } from '@chronobank/core/redux/persistAccount/utils'
+import { navigateBack } from "../../../../packages/login-ui/redux/actions";
```
Comment (wAISw) [inline src/components/login/WalletImportPage/WalletImportPage.js]: wrong path

File: src/components/operations/Operations.jsx [hunk 3]
```diff
@@ -186,7 +187,7 @@ export default class PendingOperations extends PureComponent {
 
 function mapStateToProps (state) {
   const operations = state.get('operations')
-  const network = state.get('network')
+  const network = state.get(DUCK_NETWORK)
```
Comment (ozalexo) [inline src/components/operations/Operations.jsx]: For the line 190.

Please create appropriate constant for the 'operations' duck: DUCK_OPERATIONS.
Multiple usage of 'operations' string accross the project.

File: src/components/index.js (hunk 1 above)
Comment (wAISw) [inline src/components/index.js]: wrong path

File: src/components/login/WalletImportPage/WalletImportPage.js (hunk 2 above)
Comment (ozalexo) [inline src/components/login/WalletImportPage/WalletImportPage.js]: import { navigateBack } from "@chronobank/login-ui/redux/actions"

Also do not use ";"

File: src/components/dashboard/MicroIcon/QRIcon.jsx [hunk 4]
```diff
@@ -69,12 +69,11 @@ export default class QRIcon extends PureComponent {
 
         </span>
         <Popover
-          zDepth={3}
           open={this.state.isQROpen}
           anchorEl={this.state.qrAnchorEl}
           anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
-          targetOrigin={{ horizontal: 'right', vertical: 'top' }}
-          onRequestClose={() => this.handleQRClose()}
+          onClose={() => this.handleQRClose()}
```
Comment (wAISw) [inline src/components/dashboard/MicroIcon/QRIcon.jsx]: remove arrow function.
eslint error

File: src/components/login/CreateAccountPage/CreateAccountPage.js [hunk 5]
```diff
@@ -38,8 +38,6 @@ class CreateAccountPage extends PureComponent {
   }
 
   static propTypes = {
-    previousPage: PropTypes.func.isRequired,
-    nextPage: PropTypes.func.isRequired,
```
Comment (ozalexo) [inline src/components/login/CreateAccountPage/CreateAccountPage.js]: navigateToSelectWallet is not used here also.

File: src/components/dashboard/MicroIcon/QRIcon.jsx (hunk 4 above)
Comment (ozalexo) [inline src/components/dashboard/MicroIcon/QRIcon.jsx]: It is better to not use fat-arrow functions in JSX.

Constructor:
```
  constructor (props) {
    super(props)
    this.state = {
      isQROpen: false,
      qrData: null,
      qrAnchorEl: null,
    }
    this.handleQRClose = this.handleQRClose.bind(this)
  }
```

And here:
```
onClose={this.handleQRClose}
```

File: packages/login-ui/components/NetworkCreateModal/NetworkDeleteModal/NetworkDeleteModal.js [hunk 6]
```diff
@@ -44,7 +44,7 @@ export default class NetworkCreateModal extends PureComponent {
     network: null,
   }
 
-  handleDeleteNetwork(){
+  handleDeleteNetwork (){
```
Comment (ozalexo) [inline packages/login-ui/components/NetworkCreateModal/NetworkDeleteModal/NetworkDeleteModal.js]: Please enable ESLint.
`handleDeleteNetwork (){` -> `handleDeleteNetwork () {`
Such things are highlighted by any linters.

File: packages/login-ui/components/LoginWithPrivateKey/LoginWithPrivateKey.js [hunk 7]
```diff
@@ -9,6 +9,7 @@ import { reduxForm, Field } from 'redux-form/immutable'
 import { Translate } from 'react-redux-i18n'
 import { TextField } from 'redux-form-material-ui'
 import Button from 'components/common/ui/Button/Button'
+import AccountProfileModel from '@chronobank/core/models/wallet/persistAccount/AccountProfileModel'
```
Comment (ozalexo) [inline packages/login-ui/components/LoginWithPrivateKey/LoginWithPrivateKey.js]: AccountProfileModel is not used in this file

File: packages/login-ui/components/LoginWithMnemonic/LoginWithMnemonic.js [hunk 8]
```diff
@@ -8,6 +8,7 @@ import React, { PureComponent } from 'react'
 import { reduxForm, Field } from 'redux-form/immutable'
 import { TextField } from 'redux-form-material-ui'
 import { Translate } from 'react-redux-i18n'
+import spinner from 'assets/img/spinningwheel-1.gif'
```
Comment (ozalexo) [inline packages/login-ui/components/LoginWithMnemonic/LoginWithMnemonic.js]: spinner is not used in this file

File: packages/login-ui/components/LoginWithLedger/LoginWithLedger.js [hunk 9]
```diff
@@ -81,11 +82,11 @@ class LoginLedger extends PureComponent {
     this.props.startLedgerSync()
   }
 
-  componentWillReceiveProps ({ ledger }) {
-    if (!ledger.isFetched && !ledger.isFetching && ledger.isHttps && ledger.isU2F && ledger.isETHAppOpened) {
+  componentDidUpdate (prevProps) {
```
Comment (ozalexo) [inline packages/login-ui/components/LoginWithLedger/LoginWithLedger.js]: Please use Enter button! ;-)
Also I suggest a little improvment:
```
  componentDidUpdate (prevProps) {
    const ledger = this.props.ledger
    if (!ledger.isFetched && !ledger.isFetching && tledger.isHttps && ledger.isU2F && ledger.isETHAppOpened) {
      this.props.fetchAccount()
    }
    ledgerProvider.setWallet(prevProps.account[0])
    networkService.selectAccount(prevProps.account[0])
    networkService.setAccounts(prevProps.account)
  }
```

**Pull request merged**